### ISSUE TITLE
ANDI's Output different from the screen reader's: a reason is missing from answers listed

### ISSUE THREAD

**Issue by spanchang** [SSAgov/ANDI#267]
Refer to https://www.ssa.gov/accessibility/andi/help/faq.html
Q. Why is ANDI's Output different from the screen reader's?
Comment:
One reason that is mentioned in the tutorial is omitted from the answers listed against this question: 
If the accessible name for button contains the word  "button", a screen reader appears to speak that word twice. In fact the screen reader is doing its job as expected: voice the role. The accessibility name in fact is the culprit which should exclude the role from the accessible name.
Thanks,
Sailesh
